src/process/enumeration.rs

```diff
@@ -0,0 +1,43 @@
+use rayon::iter::{IntoParallelRefIterator, ParallelIterator};
+
+use crate::api_map::APIMap;
+
+#[derive(Debug, Clone)]
+pub struct EnumerationMember {
+    pub identifier: String,
+    pub value: Option<String>,
+}
+
+#[derive(Debug, Clone)]
+pub struct Enumeration {
+    pub identifier: Option<String>,
+    pub r_type: String,
+    pub members: Vec<EnumerationMember>,
+}
+
+pub fn enumeration_processor(api_map: &APIMap) -> Vec<Enumeration> {
+    let enumerations: Vec<Enumeration> = api_map
+        .enums
+        .clone()
+        .par_iter()
```

Comment:
.par_iter() imo makes sense only in code where each iteration takes significant amount of time, like in the namespace generator with regexes. This bit right here will probably be fine with a simple .into_iter()